documentation/docs/getting_started/installation/docker_compose/docker_compose.md

```diff
@@ -49,6 +49,12 @@ newly created `.env` and `mongo-init.js` files accordingly.
 Important The `server-secret` must be 32 characters length
 :::
 
+
+> You can create a server-secret using the following command:
```

Comment:
```suggestion
:::tip You can create a server secret using the following command 
```

Reply:
In case you want to use an admonition like the one below.  If not please ignore. 

![image](https://user-images.githubusercontent.com/85233773/151145144-536d0791-fd69-4700-b17a-69884d473b7e.png)